.gitattributes

```diff
@@ -1,8 +1,8 @@
 /tests export-ignore
 /.gitattributes export-ignore
+/.github export-ignore
 /.gitignore export-ignore
-/.php_cs.dist export-ignore
-/.styleci.yml export-ignore
-/.travis.yml export-ignore
+/.php-cs-fixer.dist.php export-ignore
+/phpstan.neon export-ignore
 /phpunit.xml.dist export-ignore
 /ruleset.xml export-ignore
```

Comment:
@ChristianRiesen do you know what ruleset.xml is? looks like an obsolete codestyle configuration, do we still need the file in the repository?

Reply:
I would remove it, as we moved to php-cs-fixer

Reply:
i think so. removed the file.